### DISCUSSION TITLE
rusdesk tray startup is tried via sudo (and denied) which results in an email

### DISCUSSION THREAD

**Discussion by jankatins** [rustdesk/rustdesk#9802] [General]
### Bug Description

I recently enabled emails from my local laptop and now I get the following email when I start up:

```
Debian-gdm : user NOT in sudoers ; PWD=/ ; USER=Debian-gdm ; ENV=XDG_RUNTIME_DIR=/run/user/116 ; 
COMMAND=/usr/lib/rustdesk/rustdesk --tray
```

I do have a tray icon of rustdesk when I login to my computer. No idea what tried to sudo, though.

### How to Reproduce

Install rustdesk via a deb on debian unstable

I guess: enable start to tray somehow (no idea, too long ago)

### Expected Behavior

sudo is not used by a startup script or without my knowledge/explicit ok

### Operating system(s) on local side and remote side

debian unstable

### RustDesk Version(s) on local side and remote side

1.3.1

### Screenshots

--

**Comment by smarsching** [comment]
We are also experiencing this problem. I believe that this is a bug in RustDesk and have filed an [issue](https://github.com/rustdesk/rustdesk/issues/11554).

**Comment by rustdesk** [comment]
> I think that run_as_user should not be used at all, because the server process is already running as the correct user and the tray process is only spawned from this process. The latter is ensured by the surrounding if block starting in line 1859.

Yes, you are correct. It was root in old version, we forget to change this.

**Reply by rustdesk** [reply]
https://github.com/rustdesk/rustdesk/commit/5c2538e7af2c2fba3a0cd6acffc75ef52c5bd734

**Reply by smarsching** [reply]
Great, thanks for fixing this so quickly.
